.github/workflows/ci.yml

```diff
@@ -26,20 +26,16 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest, macOS-latest]
-        node: [13.x, 12.x, 10.x, "10.12.0"]
+        node: [14.x, 12.x, 10.x, "10.12.0"]
```

Comment:
Our supported Node.js versions are `^10.12.0 || >=12.0.0`. Therefore, it's too early to remove `13.x`.

Reply:
It's common to only test `even-numbered` versions (unless latest release is `odd-numbered`).

No body test `11.x`

Reply:
> No body test `11.x`

Yes, our supported Node.js versions are `^10.12.0 || >=12.0.0`.

Reply:
Right.

Reply:
As per https://github.com/nodejs/Release#release-schedule, Node.js `13.x` will be EOL on June 2020. ESLint will drop the support of `13.x` at a major version after the EOL. Thet's the proper time to remove `13.x` tests, I think.

Reply:
https://github.com/eslint/eslint/pull/13210/commits/082588ed721f6fb9d72e672b4d3994fb2cf1456a

`14.x, 13.x, 12.x, 10.x, "10.12.0"` - linux
`12.x` only - windows/mac

Reply:
Great, thank you!